src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,52 @@
+package duke;
+
+import duke.exceptions.DukeException;
+import duke.handlers.DukeCommand;
+import duke.models.Parser;
+import duke.models.Storage;
+import duke.models.TaskList;
+import duke.models.Ui;
+
+import java.util.Scanner;
+
+public class Duke {
+    public static TaskList taskList;
+    public Ui ui = new Ui();
+    public Parser parser = new Parser();
+
+    public Duke() {
+        try {
+            taskList = Storage.loadTasksFromDisk();
+        } catch (DukeException e) {
+            taskList = new TaskList();
+        }
+    }
+
+    public void chat (Scanner sc) {
```

Comment:
Might want to remove the whitespace between chat and the arguments
```suggestion
    public void chat(Scanner sc) {
```